src/Rector/Deprecation/StaticArgumentRenameBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Renames deprecated the argument in a static call.
+ */
+abstract class StaticArgumentRenameBase extends AbstractRector {
```

Comment:
This is the general use base class.

Reply:
The bases classes are great. It took me a while to realize they existed; they get lost in between the non-base classes. I didn't realize until now that class names ending with *Base.php were all the re-usable utility classes and that I could find them by looking at the class names. D'oh! One developer experience (DX) improvement could be to put them in their own sub-package.